.github/workflows/linting.yml

```diff
@@ -25,3 +25,7 @@ jobs:
         flake8 --exclude=\.eggs,tests,docs,config/Dockerfiles --ignore=E124,E303,W504 --max-line-length 80 .
         #flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics;
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+    - name: "Execute linting for AWS role"
+      run: |
```

Comment:
@14rcole All workflows are independent executors. 
you will have to run make sure the container has molecule installed before you run any of the molecule commands.

Reply:
Got it, thanks